contracts/VestingSimple.sol

```diff
@@ -35,68 +35,65 @@ contract VestingSimple is DSMath, DSAuth {
   uint256 public vestingDuration;
   uint256 public startTime;
 
+  uint256 public totalGrants;
+  uint256 public totalClaimed;
```

Comment:
These are just for the UI? I was expecting an assert somewhere using these when I saw they'd been added.

Reply:
They were but no reason not to add an assertion.